documentation/staging/content/managing-domains/accessing-the-domain/status-conditions.md

```diff
@@ -195,15 +195,22 @@ The following is a list of condition types for a Cluster resource.
   * There are no pending server shutdown requests.
 
 #### `Available` {id="cluster-available"}
-- The `status` attribute is set to  `True` when a sufficient number of WebLogic Server pods are
-  ready in the cluster. This includes either:
-  * All WebLogic Server pods in the cluster are ready, as configured in the `cluster.spec.replicas` field,
-    or if it is not configured, then in the `domain.spec.replicas` field.
-  * Some of the expected server pods for the cluster are temporarily not ready, and the number of
-    `not ready` server pods is less than or equal to `cluster.spec.maxUnavailable` which defaults to `1`.
-- The `status` attribute is also set to `True` when no WebLogic Server pods are running and this
-  is the expected state, such as when `cluster.spec.replicas` is set to `0`, or when
-  `cluster.spec.serverStartPolicy` is set to `Never`.
+- The `status` attribute is set to `True` when a sufficient number of WebLogic Server pods are
+  ready in the cluster. Both of the following must be true:
+  - At least one pod in the cluster is expected to run and is `ready`.
+  - The number of `not ready` server pods which are expected to run
+    is less than or equal to `cluster.spec.maxUnavailable` which defaults to `1`.
+- Examples:
+  - If a cluster has `serverStartPolicy` `Never` or `replicas` `0`,
+    or a cluster is in a domain with `serverStartPolicy` `AdminOnly` or `Never`,
+    then the cluster will have `Available` `False`.
```

Comment:
Re-write (I can hardly understand the meaning of this sentence, but here's a stab at it): If a cluster has a `serverStartPolicy` of `Never`, (comma) or `replicas` is `0`, or a cluster is in a domain with a `serverStartPolicy` of `AdminOnly` or `Never`, (comma) then the cluster will be `Available`, `False`.